### ISSUE TITLE
use slingshot with AWS China (the region is cn-north-1)

### ISSUE THREAD

**Issue by irrigator** [CulturalMe/meteor-slingshot#128]
I want to use Slingshot to upload files to S3. I work with the region cn-north-1 of AWS China. AWS China requires a different set of credentials from other regions in the world and I can't find cn-north-1 in the list given by the README.md.

Any idea on how can I use Slingshot with this specific region?

**Comment by irrigator** [comment]
I just found this code snippet in the source code of Slingshot. I guess it works well with cn-north-1, then.
```
if (region === "cn-north-1")
  bucketDomain = "s3.cn-north-1.amazonaws.com.cn";
```

**Comment by gsuess** [comment]
That part is not released yet. I'll release this along with some other features.

**Comment by irrigator** [comment]
Got it. Does it mean right now I need to override the bucketUrl by hand?

Thanks for your time.

**Comment by gsuess** [comment]
Yes, that's correct.

**Comment by jonycfu** [comment]
Note to future China region users at time of post writing: 

**Configuring `region` with `"cn-north-1"` and a `bucket` name will return the wrong URL from `createDirective`**

Without hardcoding `bucketUrl` option, the following url is returned:
`https://my_bucket_name.s3-cn-north-1.amazonaws.com/`, 
which causes a `net::ERR_NAME_NOT_RESOLVED` Error. 

So what I did was added in `bucketUrl` with the following override: 
`bucketUrl: 'https://s3.cn-north-1.amazonaws.com.cn/my_bucket_name'` 

This, will correctly reference the correct URL. Not sure if I'm misusing the package or china region is not supported yet?
